inclusion/language.go

```diff
@@ -48,8 +48,8 @@ var inclusiveFilters = []InclusiveFilter{
 	{Filter: "retraso mental", Reply: "*Retrasado* y *Retraso mental* son términos despectivos eliminados del vocabulario psiquiátrico y, según la OMS, la forma correcta para referiste a ese grupo de enfermedades y transtornos es *Trastorno del desarrollo intelectual* "},
 	
 	// Our own list
-	{Filter: "gentlemen", Reply: "Instead of *gentlemen*, perhaps you mean *folks*?... *[Please consider editing your message so it's more inclusive]*"},
-	{Filter: "ladies", Reply: "Instead of *ladies*, perhaps you mean *folks*?... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "gentlem(a|e)n", Reply: "Instead of *gentlem(a|e)n*, perhaps you mean *folks*?... *[Please consider editing your message so it's more inclusive]*"},
+	{Filter: "lad(y|ies)", Reply: "Instead of *lad(y|ies)*, perhaps you mean *folks*?... *[Please consider editing your message so it's more inclusive]*"},
```

Comment:
I think that the plural forms are the ones that we want to avoid (in particular, "gentlemen"), as they can be used to incorrectly refer to a group in a non-inclusive way.

These singular forms "gentleman" and "lady" are rarely used without an article "a/the" before them, and in such cases, the suggested replacement "folks" doesn't fit. If we want to avoid referring to people as "a lady" or "a gentleman", then a more appropriate generic replacement may be just "a person".

But then again, do we actually want to filter on these words? If we did that, wouldn't we also want to avoid "boy/girl", "man/woman", etc. as well?

Reply:
Thanks for your feedback! Good arguments in here 🙌

Candebot does not truly filter words but just tells you that, perhaps, you meant another word based on a list of words that usualy are included in a context where inclusivity is violated.
The message is only visible by the user who wrote that particular word. It uses a neutral language as well.

I agree the words boy/girl, among others, would be also need to be considered.

As conclusion, considering the fact that the message Candebot writes you is ephemeral and not public, i think we have more chances to have a positive impact than a negative.

Wdyt?

Reply:
Right, "filter" was a poor choice of word on my part. I just meant that the words would be matched by one of the patterns.

I must admit that I have never seen the bot in action, so I'm not sure what the UX is like, but if it is not too intrusive then I guess that it can be considered a "net positive" to include more words. So maybe just changing the recommendation for the singular "lady/gentleman" to "person" would be my only suggestion 👍